docs/views/install.html

```diff
@@ -28,7 +28,9 @@ <h1 class="govuk-heading-xl">Installing the kit</h1>
 
       <h2 class="govuk-heading-m">1. Download the kit</h2>
 			<p>
-	      <a href="/docs/download" data-link="download">Download (zip)</a>
+	      <a href="https://github.com/alphagov/govuk-prototype-kit/archive/{{ releaseVersion }}.zip" data-link="download">Download {{ releaseVersion }} (zip)</a>
+        </br>
+        <a href="https://github.com/alphagov/govuk-prototype-kit/releases/{{ releaseVersion }}">Read the release notes for {{ releaseVersion }}</a>
```

Comment:
This is temporary, will figure out the best way to communicate the release notes and what you can do with them with Mark.